regolith/schemas.py

```diff
@@ -3620,6 +3621,172 @@
             "type": "boolean",
         },
     },
+    "projecta": {
+        "_description": {
+            "description": "This collection describes a single deliverable "
+                            "of a larger project."
+        },
+        "_id": {
+            "description": "Unique projecta identifier",
```

Comment:
the singular of projecta is projectum.  This will need to be changed nearly everywhere...